recipes/bqplot/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.6.1" %}
+
+package:
+    name: bqplot
+    version: {{ version }}
+
+source:
+    fn: bqplot-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/b/bqplot/bqplot-{{ version }}.tar.gz
+    md5: e0ab4e1f478b85c59e774eb58317cd67
+
+build:
+    number: 1
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
```

Comment:
I looked closer and it seems the answer is that it is a dependency, but we are not linking to it. So, this is fine.